app/src/main/java/net/squanchy/notification/NotificationAlarmReceiver.kt

```diff
@@ -0,0 +1,20 @@
+package net.squanchy.notification
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+
+import timber.log.Timber
+
+class NotificationAlarmReceiver : BroadcastReceiver() {
+
+    override fun onReceive(context: Context, intent: Intent) {
+        if (Intent.ACTION_BOOT_COMPLETED != intent.action) {
+            Timber.d("Intercepted spoofed intent directed to NotificationAlarmReceiver: %s", intent)
```

Comment:
we might want to do this with Kotlin string interpolation: `"Intercepted spoofed intent directed to NotificationAlarmReceiver: $intent"`. No big deal, but probably it just reads better and looks more idiomatic